.gitignore

```diff
@@ -1,4 +1,5 @@
 build*/
+cmake-build*/
```

Comment:
can't see this mentioned anywhere else?

Reply:
Ah that snuck in by mistake, it's just where CLion puts build directories by default and not really anything to do with this PR.